CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v1.7.2-rc1
+
+### Note
+
+- A new flag `changeRequestLimit` is added to the Kyverno main container, this flag sets the up-limit of reportchangerequests that a namespace can take, or clusterreportchangerequests if matching kinds are cluster-wide resources. The default limit is set to 1000, and it's recommended to configure it to a small threshold on large clusters. Here the large clusters are considered that a policy report has more than 1k results. 
```

Comment:
would it be better to name as `maxReportChangeRequests`?

Reply:
I avoid using `ReportChangeRequest` as this also restricts ClusterReportChangeRequest count.

Reply:
How about `maxChangeRequests`?